packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -338,28 +344,31 @@ export default class NewVm extends BaseComponent {
 
   // Actions ---------------------------------------------------------------------
 
-  _reset = () => {
-    this._replaceState({
-      bootAfterCreate: true,
-      CPUs: '',
-      cpuCap: '',
-      cpuWeight: '',
-      existingDisks: {},
-      fastClone: true,
-      hvmBootFirmware: '',
-      installMethod: 'noConfigDrive',
-      multipleVms: false,
-      name_label: '',
-      name_description: '',
-      nameLabels: map(Array(NB_VMS_MIN), (_, index) => `VM_${index + 1}`),
-      namePattern: '{name}%',
-      nbVms: NB_VMS_MIN,
-      VDIs: [],
-      VIFs: [],
-      seqStart: 1,
-      share: false,
-      tags: [],
-    })
+  _reset = (callback = noop) => {
```

Comment:
`noop` is unnecessary, `undefined` is perfectly supported by `setState` (and `_replaceState`).